app/translations/messages.de_DE.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Your inquiry at Wikimdia": "Ihre Anfrage an Wikimedia",
```

Comment:
I generally like the idea and I am not familiar with this Symfony Translator but two minor remarks (just for you to possibly consider, I do not have a strong opinion about those things):
 - this "dictionary format" of `source_message: translated_message` makes sense but my only concern would be that when there is a need to change a subject of an email, you'd probably need to change the source part in all message files not only the translation part (plus in places where the particular translation is called), so you don't end up with things like `"Your inquiry at Wikimedia": "Your email to Wikimedia"`. Mediawiki stuff uses some more abstract things as keys, e.g. it'd be `"inquiry-email-subject": "Your inquiry at Wikimedia"`. This approach has this drawback that you have to look up what is the actual message (ie. what `trans("inquiry-email-subject")` stands for). But changing translation then only means changing "values" in the message files. I don't know which approach would fit better this use case. There shouldn't not be that much translatable messages here, and changes haven't been often.
 - I wouldn't expect it makes much of difference in this fundraising app but why having locale-specific message files and not language-specific? I doubt there will be different transations for Swiss German locale, British English etc.

Oh, and there is an "e" missing in "Wikimedia".